README.md

```diff
@@ -31,6 +31,12 @@ KFD uses an **un-distribution model**. This means that we:
 
 ## Architecture 🏗
 
+<!-- markdownlint-disable MD033 -->
+<p align="center">
+  <img src="docs/assets/kfd-architecture.png" width="800px"/>
```

Comment:
this file should be backported to this branch also

Reply:
done ✅